server/BraveAlerterConfigurator.js

```diff
@@ -96,12 +75,12 @@ class BraveAlerterConfigurator {
       session = await db.getSessionWithSessionId(alertSession.sessionId, pgClient)
 
       if (session) {
-        // If this is not a OneSignal session (i.e. we are given a respondedByPhoneNumber) and the session has no respondedByPhoneNumber, then this is the first SMS response, so assign it as the session's respondedByPhoneNumber
+        // If the session has no respondedByPhoneNumber, then this is the first SMS response, so assign it as the session's respondedByPhoneNumber
         if (alertSession.respondedByPhoneNumber !== undefined && session.respondedByPhoneNumber === null) {
```

Comment:
I think that we don't need the `alertSession.respondedByPhoneNumber !== undefined` part of this condition because from now on, we can assume that we are using SMS and there will be an `alertSession.respondedByPhoneNumber` passed to this function every time. What do you think?